lib/upipe-av/upipe_avcodec_decode.c

```diff
@@ -1324,6 +1324,13 @@ static void upipe_avcdec_output_pic(struct upipe *upipe, struct upump **upump_p)
     if (side_data)
         uref_pic_set_cea_708(uref, side_data->data, side_data->size);
 
+#ifdef AV_FRAME_DATA_S12M_TIMECODE
```

Comment:
Can anyone explain how this #ifdef can fail with an ffmpeg version new enough to have the side data?

Reply:
`AV_FRAME_DATA_S12M_TIMECODE` is an enum value, not a preprocessor define. To check for availability, you can use `#if LIBAVUTIL_VERSION_INT >= AV_VERSION_INT(56, 20, 100)`.

Reply:
Yes of course!  How dumb of me.  Is a check really needed?  What minimum version is assumed elsewhere?  I'll look for some other checks that already exist and try to judge.

Reply:
S12M was added in 4.1 and there's a check for libavcodec version available in 4.0 so I will use your check.  Thank you.